app/models/operatingsystems/solaris.rb

```diff
@@ -41,8 +35,12 @@ def url_for_boot(file)
     pxedir + "/" + PXEFILES[file]
   end
 
+  def available_loaders
+    # provide a dummy value as pxe_loader is ignored for this OS
+    { "Solaris" => "UNUSED" }
```

Comment:
Wouldn't it be better to just hide the pxe_loader field in the UI if this is an empty hash?